### ISSUE TITLE
Please give me the pretrained model's curl link.

### ISSUE THREAD

**Issue by bemoregt** [akanimax/pro_gan_pytorch#20]
Hi, @joebradly @joshalbrecht @akanimax @SaoYan 

Please give me the pretrained models curl link for terminal.

$ curl -O URL 

My terminal cannot download from your google drive site.

Thanks in advance ~

**Comment by akanimax** [comment]
Hi @bemoregt,

Please try the following link for downloading the best model for the `Demo.py` script.
[https://doc-10-44-docs.googleusercontent.com/docs/securesc/ha0ro937gcuc7l7deffksulhg5h7mbp1/onshn1ab5inp66l8mio0c1c4b6ld1tol/1552456800000/14224252584598248560/*/1riExY5yWZpOMMhdMLVnKXiJbp8mPHsUx?e=download](https://doc-10-44-docs.googleusercontent.com/docs/securesc/ha0ro937gcuc7l7deffksulhg5h7mbp1/onshn1ab5inp66l8mio0c1c4b6ld1tol/1552456800000/14224252584598248560/*/1riExY5yWZpOMMhdMLVnKXiJbp8mPHsUx?e=download)

Basically, start downloading the files in a browser and then pause it. Copy the link from there to use it with curl. I usually do this. 

Hope it helps!

Cheers :beers:!
@akanimax

**Comment by bemoregt** [comment]
Thanks ~!

**Comment by akanimax** [comment]
You are welcome :smile:! 
Closing the issue now.
